### PR TITLE
Initial draft of 3.4.0 release blog post.

### PR DESCRIPTION
**Pull request by myronmarston**
I think this is ready for review.  I plan to publish it in the morning.

/cc @rspec/rspec

### CODE REVIEW

**Comment by myronmarston** [comment]
Actually, I'm not sure I can publish right now...I've never worked out my deploy woes :(.

File: source/blog/2015-11-13-rspec-3-4-has-been-released.md [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+
+RSpec finds the failure code snippet by looking through the exception
+stack trace for an appropriate frame. We could just use the top stack
+frame but that's generally not what you want: when you have an expectation
+failure, the top frame refers to a link in RSpec where the
+`RSpec::Expectations::ExpectationNotMetError` was raised, and you want
+to see the snippet from your `expect` call site rather than seeing
+a snippet of RSpec's code. Before RSpec 3.4, our solution for this
+was fairly naive: we just looked for the first
+stack frame from your spec file. In some situations this would display
```
Comment (yujinakayama) [inline source/blog/2015-11-13-rspec-3-4-has-been-released.md]: Maybe we should say something like "your spec file the current running example is in"? I think It's unclear if a helper file in `spec/support` is a _spec file_ or not.

File: source/blog/2015-11-13-rspec-3-4-has-been-released.md [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+### Expectations: Better Compound Failure Messages
+
+Continuing with the "improved failure output" theme, rspec-expectations
+3.4 provides better failure messages for compound expectations. Before,
+we would combine each failure message into a single line. For example,
+this expectation:
+
+~~~ ruby
+expect(lyrics).to start_with("There must be some kind of way out of here")
+              .and include("No reason to get excited")
```
Comment (yujinakayama) [inline source/blog/2015-11-13-rspec-3-4-has-been-released.md]: Not aligned

Comment (myronmarston) [inline source/blog/2015-11-13-rspec-3-4-has-been-released.md]: It's not clear to me what's the best way to align this.  I aligned `.to` and `.and` but I'm open to an alternate suggestion.  How would you format this?

File: source/blog/2015-11-13-rspec-3-4-has-been-released.md (hunk 1 above)
Comment (myronmarston) [inline source/blog/2015-11-13-rspec-3-4-has-been-released.md]: Good point.  I've edited.  Let me know what you think.

BTW, I also added a note about the `config.max_displayed_failure_line_count` option -- thoughts?

**Comment by xaviershay** [comment]
LGTM

**Comment by myronmarston** [comment]
Can someone deploy this?  Deploys still don't work for me :(.

**Comment by yujinakayama** [comment]
Done.

```
$ TARGET=prod bundle exec middleman deploy

...

      create  /Users/nkymyj/Projects/rspec.github.io/build/mstile-144x144.png
      create  /Users/nkymyj/Projects/rspec.github.io/build/mstile-150x150.png
      create  /Users/nkymyj/Projects/rspec.github.io/build/mstile-310x310.png
== Locales: en, ja (Default en)
== Locales: en, ja (Default en)
== Locales: en, ja (Default en)
## Deploying via git to remote="git@github.com:rspec/rspec.github.io.git" and branch="master"
Already on 'master'
[master a2bd74e] Automated commit at 2015-11-14 03:59:45 UTC by middleman-deploy 1.0.0
 1 file changed, 1 insertion(+)
 create mode 100644 CNAME
Counting objects: 2098, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (1951/1951), done.
Writing objects: 100% (2098/2098), 2.93 MiB | 1.36 MiB/s, done.
Total 2098 (delta 1543), reused 0 (delta 0)
To git@github.com:rspec/rspec.github.io.git
 + f29bceb...a2bd74e master -> master (forced update)
```

**Comment by myronmarston** [comment]
Thank you, @yujinakayama!

**Pull request merged**